_pages/documentation/gem5-stdlib/5-local-resources-support.md

```diff
@@ -0,0 +1,161 @@
+---
+layout: documentation
+title: Local Resources Support in gem5
+parent: gem5-standard-library
+doc: gem5 documentation
+permalink: /documentation/gem5-stdlib/local-resources-support
+author: Kunal Pai, Harshil Patel
+---
+
+# Adding a Workload to gem5
```

Comment:
Either this should be an `h2` (i.e., `##`) or you can skip this since you already have a title.